### DISCUSSION TITLE
MariaDB supported?

### DISCUSSION THREAD

**Discussion by macidev** [Leantime/leantime#1782] [Q&A] [answered]
Hi, 

Just a quick question. Is MariaDB supported as well? In the requirements is only MySQL mentioned.

Since MySQL and MariaDB are basically drop-in replacements, but in details different, I'm curious. 

Thanks for the answer.

**Comment by jphovila** [comment]
Leantime appears to work just fine with MariaDB.

As would basically any software designed to work with MySQL. I assume there are some that actually don't. Never bumped into one myself, yet.

**Comment by macidev** [comment]
Hey @jphovila,
Thanks for your reply. 

Basically MySQL and MariaDB are in theory until MySQL 8 "the same". BUT, running a software on a MariaDB Server, because MySQL is not available needs some clarification for the future. I do not want to run a software which won't support MariaDB.

**Comment by marcelfolaron** [comment, marked as answer]
Hi Marcus,

We are supporting MariaDB, in fact our cloud instances are running on
Maria. While we are testing both MariaDB and Mysql we rarely use any
“exotic” queries. Most of our db queries are fairly standard SQL that are
supported on almost all platforms.

**Reply by mariadb-pieterhumphrey** [reply]
@marcelfolaron would dbgate be interested in collaborating to update the connector software, presuming that's the issue behind bugs like this one?  https://github.com/dbgate/dbgate/issues/590.

**Comment by BernhardBaumrock** [comment]
Would be nice to add MySQL/MariaDB to the list of requirements :) @marcelfolaron

**Comment by marcelfolaron** [comment]
I’ll make sure that gets added.
What is dbgate? Wrong repo? I’d be happy to discuss connector abstraction
though. We’ve had a couple requests for Postgres.

**Reply by mariadb-pieterhumphrey** [reply]
Apologies @marcelfolaron yes wrong repo, sorry about that.

**Reply by macidev** [reply]
Postgres would be awesome. We use Postgres in out company a lot and would love to use leantime with PostgreSQL as well.
